apps/game_backend/lib/game_backend/items/item_cost.ex

```diff
@@ -0,0 +1,26 @@
+defmodule GameBackend.Items.ItemCost do
+  @moduledoc """
+  Contains a set the different prices to purchase an ItemTemplate.
+  """
+  alias GameBackend.Users.Currencies.CurrencyCost
+  alias GameBackend.Items.ItemTemplate
+
+  use GameBackend.Schema
+  import Ecto.Changeset
+
+  schema "item_costs" do
+    field(:name, :string)
+    belongs_to(:item_template, ItemTemplate)
+    embeds_many(:currency_costs, CurrencyCost)
```

Comment:
Let's rename currency_costs to something like purchase_cost so that it's more descriptive!

Reply:
Sure thing!